packages/react/etc/react.api.md

```diff
@@ -9462,7 +9462,7 @@ export interface ITooltipHost {
 }
 
 // @public
-export interface ITooltipHostProps extends React_2.HTMLAttributes<HTMLDivElement | TooltipHostBase> {
+export interface ITooltipHostProps extends Omit<React_2.HTMLAttributes<HTMLDivElement | TooltipHostBase>, 'content'> {
```

Comment:
this API change is actually a noop from consumers POV as `content` is presented within the interface definition which is the source of truth (previously it would override the base content from HTMLAttributes interface)